### ISSUE TITLE
Create Performance Test Plan

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#39621]
## Questions to Answer
- [x] How many appointments per day will there by at VISN 8 facilities?
- [x] What is the anticipated load for CI & PCI for first week in May (variability by close to top of hour)?
- [x] When do PCI reminders go out?
- [x] Which components & subsets of components are we going to target?
- [x] With which environment can we do load testing?
- [x] Do we need approval(s) to perform load testing?
- [x] How are going to set up the data needed for appointments?
- [x] What test accounts are we going to use?
- [x] Do we use CHIP and LaROTA in VistA or do we mock them?
- [x] For Stephen: Should each team test independently or should we load test collaboratively? (Adrian starting a thread)
- [x] How are we going to test? Hitting each endpoint or complete end-to-end.

## AC
- [x] Get answers to the above questions and document
- [x] Create the performance test plan
- [x] Present to team

## Next Steps
Translate estimated load to difference subsystem calls

**Comment by randomsync** [comment]
test plan draft here: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin/engineering/performance-testing

**Comment by randomsync** [comment]
test plan is published: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin/engineering/performance-testing. I have identified the subsystems and we have initiated conversations with CHIP team to collaborate on the effort. 

Test data (appointments) for testing backend subsystems will be created using Shane's scripts on EC2 instances in VAEC account. We're most likely going to use k6 for test script creation and load generation, and use DataDog to monitor resource utilization during the tests.
